scripts/github-status/github-status.rb

```diff
@@ -84,7 +84,7 @@ def print_pr_sha_info(pull)
         print_pr_sha_info(p)
       end
     else
-      puts "#{pull.number}:#{pull.head.sha}" if pull
+      puts "#{pull.number}:#{pull.head.sha}:#{pull.base.ref}" if pull
```

Comment:
this is the string that ends up in the description right?
is there a way to trim the `sha` a bit here? maybe to 7, as done e.g. by `git rev-parse --short HEAD`